vfh:
  LA: 100
  wg: 1
  wd: 1
  wo: 15
  crit_cost: 1
  ndpsi: 360
